static/js/message_fetch.js

```diff
@@ -283,7 +292,13 @@ exports.maybe_load_older_messages = function (opts) {
 
 exports.do_backfill = function (opts) {
     const msg_list = opts.msg_list;
-    const anchor = exports.get_backfill_anchor(msg_list).toFixed();
+    let anchor = exports.get_backfill_anchor(msg_list);
+    if (anchor === undefined) {
+        // Msg list is empty, start by fetching the oldest message
```

Comment:
I feel like this doesn't make any sense, shouldn't we be fetching from the newest in this case too?

Reply:
I think `do_backfill` won't be called with an empty message list.  

I'm starting to think we just want a `blueslip.fatal` in `get_backfill_anchor` / `get_frontfill_anchor` for cases that can't happen -- that'll avoid this sort of confusing conditional for an impossible case.

Reply:
Yeah, the fact that we are here would mean our local data isn't appropriate,  and we should show a fatal error to imply it correctly.